src/server.ts

```diff
@@ -0,0 +1,45 @@
+import * as express from "express";
```

Comment:
I saw linting configured for single quotes. Please make it consistent. Running linting before final review will be good idea until we have CI doing it for us.